src/mixed-line-bar-chart/utils.ts

```diff
@@ -130,3 +114,43 @@ export function isDataSeries<T>(
 ): series is MixedLineBarChartProps.DataSeries<T> {
   return series.type === 'line' || series.type === 'bar';
 }
+
+// Source https://stackoverflow.com/a/65186378/16648714
+export function createOneSideRoundedRectPath(
```

Comment:
The external code should live in a separate file without our standard header: "Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved."

Because technically it is not ours

See the example I shared before: https://github.com/cloudscape-design/components/blob/main/pages/utils/pseudo-random.ts